_includes/maintenance-banner-service-disruption.html

```diff
@@ -0,0 +1,15 @@
+<section aria-label="Site alert" class="usa-site-alert usa-site-alert--emergency">
+  <div class="grid-container padding-0">
+    <div class="usa-alert">
+      <div class="usa-alert__body">
+        <h3 class="usa-alert__heading text-white">
+          Service disruption
+        </h3>
+        <p class="usa-alert__text">
+          date/time: We're investigating service disruption on the .gov registrar. The .gov zone and individual domains remain online. However, the registrar is running slower than usual.
```

Comment:
In this case, wouldn't we also add the same banner to manage.get.gov, since in the case of just a slow down we would be online enough to show a banner. CC: @Katherine-Osos .

Reply:
Very good point! Yes, I think this should appear on manage.get.gov, as well.

Reply:
Okay, I will admit, we have just never seen this so I don't think it's high priority. But that said it will be quick to add the same banner to manage.get.gov if needed. @vickyszuchin could we get a ticket for having an identical banner added to manage.get.gov